moveit_core/macros/include/moveit/macros/declare_ptr.h

```diff
@@ -0,0 +1,55 @@
+/*********************************************************************
```

Comment:
+1 to `declare_ptr.h`.

The "class forward" header/macro *is* miss-named.
We *could* provide a combined header for both in kinetic and deprecate the `class_forward.h` header there,
but then I don't think it is worth the effort...